Packs/Infinipoint/ReleaseNotes/1_0_9.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Infinipoint
+- %%UPDATE_RN%%
```

Comment:
```suggestion
- Fixed an issue in command **infinipoint-execute-action** where the **target** argument did not handle single values correctly.
```
@etInfp is that the correct explanation?

Reply:
@yuvalbenshalom Thanks
update the PR